gunicorn/arbiter.py

```diff
@@ -42,10 +42,10 @@ class Arbiter(object):
     # I love dynamic languages
     SIG_QUEUE = []
     SIGNALS = [getattr(signal, "SIG%s" % x)
-               for x in "HUP QUIT INT TERM TTIN TTOU USR1 USR2 WINCH".split()]
+               for x in "CHLD HUP QUIT INT TERM TTIN TTOU USR1 USR2 WINCH".split()]
     SIG_NAMES = dict(
         (getattr(signal, name), name[3:].lower()) for name in dir(signal)
-        if name[:3] == "SIG" and name[3] != "_"
+        if name[:3] == "SIG" and name[3] != "_" and name[3:] != "CLD"
```

Comment:
This condition gave me a pause, not obvious.

```suggestion
        if name[:3] == "SIG" and name[3] != "_" and name[3:] != "CLD"  # SIGCLD is an obsolete name for SIGCHLD
```

At least according to https://www.gnu.org/software/libc/manual/html_node/Job-Control-Signals.html

Reply:
I thought I encountered an error without this (on Linux) but I'll double check.

Reply:
@sylt's solution in #3148 seems a bit cleaner.